src/compiler/reverse.jl

```diff
@@ -124,11 +124,6 @@ function instrument(ir::IR)
     ex = st.expr
     if isexpr(ex, :foreigncall, :isdefined)
       continue
-    elseif isexpr(ex, :enter, :leave)
-      error("""try/catch is not supported.
-            Refer to the Zygote documentation for fixes.
-            https://fluxml.ai/Zygote.jl/latest/limitations
-            """)
```

Comment:
I believer we still need this on versions of julia that do not have `enter` and `leave`?
so maybe we keep this and add a `&& VERSION<v"1.10"` (is that the right bounds?)